pyproject.toml

```diff
@@ -19,6 +19,7 @@ clint = "^0.5.1"
 requests-toolbelt = "^1.0.0"
 packaging = "^23.0"
 pydantic = "2.5"
+typing-extensions = "4.10.0"
```

Comment:
can we specify:
```
pydantic = “^2.5”
typing-extensions = “^4.6.1"
```
This will make poetry allow to use (eg. latest pydantic 2.6.2)
```
Requires-Dist: pydantic (>=2.5,<3.0)
Requires-Dist: typing_extensions (>=4.6.1,<5.0.0)
```